packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -321,6 +327,34 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
       startingPoint.push(prevPosition);
 
       const xValue = point.horizontalBarChartdata!.x;
+      const placeholderIndex = 1;
+
+      /**
+       * Render bar value instead of placeholder bar for relative variant
+       */
+      if (index === placeholderIndex && this.props.variant === HorizontalBarChartVariant.Relative) {
+        if (this.props.hideValues) {
+          return null;
+        }
+
+        /** Calculate bar value by subtracting placeholder value from total value */
+        const barValue = point.horizontalBarChartdata!.y - xValue;
```

Comment:
did not understand how y - x represenst barValue

Reply:
xValue of placeholder is equal to the difference between yValue and xValue of 1st bar
![image](https://user-images.githubusercontent.com/110246001/212725229-ef17eebb-5aa7-4ed8-ae54-881c817e67da.png)

Reply:
updated the statement

Reply:
Got it. For horizontal bar chart Y axis is represented in x axis scale only right?

Reply:
yes